Fix SetBaudCOM conflicting parameter types (Closes: #748036)
Patch author: Ryan Finnie <ryan@finnie.org>

Index: digitemp-3.6.0+dfsg1/userial/ds9097u/linuxlnk.c
===================================================================
--- digitemp-3.6.0+dfsg1.orig/userial/ds9097u/linuxlnk.c	2013-11-09 21:21:40.195561463 +0000
+++ digitemp-3.6.0+dfsg1/userial/ds9097u/linuxlnk.c	2014-05-13 17:55:28.847651774 +0000
@@ -105,7 +105,7 @@
 void      FlushCOM(int);
 int       ReadCOM(int, int, uchar*);
 void      BreakCOM(int);
-void      SetBaudCOM(int, int);
+void      SetBaudCOM(int, uchar);
 void      msDelay(int);
 long      msGettick(void);
 
@@ -331,7 +331,7 @@
 // PARMSET_57600    0x04
 // PARMSET_115200   0x06
 //
-void SetBaudCOM(int portnum, int new_baud)
+void SetBaudCOM(int portnum, uchar new_baud)
 {
    struct termios t;
    int rc;
